### ISSUE TITLE
Full Accessibility & 508 Office Audit [Search & Discovery- Search UI Consistency Pattern for Global Search Pages]

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#27624]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(Platform understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [ ] VFS: Using the Slack Workflow ([User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)), notify Collaboration Cycle team of this Github ticket in #vfs-platform-support. 
- [x] Platform: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)
- [ ] VFS: Close this ticket when all steps above are completed.

## Artifacts
- https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/typeahead/typeahead-staging-review-accessibility-checklist.md

> - Access information for the tool or feature in staging. Do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository
> - Link to verification that accessibility checks are complete:
>   - (preferred) Link to your [TestRail Platform accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
> OR
>   - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
> - List of known issues (with links to tickets when applicable)

## Additional information
As part of this review, Platform will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Platform Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by shiragoodman** [comment]
FYI @joshkimux @AngelaFowler82

**Comment by joshkimux** [comment]
@AngelaFowler82 and I will begin a full audit now that all staging tickets have been resolved 😄

**Comment by joshkimux** [comment]
Great work team @denisecoveyduc , we're ready to send this off to the 508 office. Sidenote: this has been one of our faster full audits in memory thanks to the thoroughness in covering all of the staging tickets prior. Special shoutout to @srsuddath in particular 🥇 

@noahgelman feel free to review as well ✌️

**Comment by denisecoveyduc** [comment]
That is fantastic news @joshkimux. @srsuddath has been instrumental - huge 🎉. Great work Sam! 

It looks r+s consistency is highlighted in most of the issues you identified.  I assigned @brianalloyd to make him and the public website team aware. 

Sam will look to address our portion of #29759 this sprint or next. 
Thanks!

**Comment by nicole-leonard** [comment]
@joshkimux Can you please take a peek at this and give a status update to myself and @AngelaFowler82 ?

**Comment by joshkimux** [comment]
@nicole-leonard will do as soon as I'm back from PTO 🙇

**Comment by joshkimux** [comment]
@nicole-leonard Checking in on the two remaining tickets, it looks like the defect-3 has been looped to @LindaLi123 and the Search and Discovery Team to resolve. The defect-4 has been addressed by @srsuddath , but it appears to have been iceboxed.

**Comment by laflannery** [comment]
Both issues are closed, closing this epic
